### ISSUE TITLE
[FEATURE] Combine Widget Dropdown With Audio Level Control Slider

### ISSUE THREAD

**Issue by PerryHugh** [kgshank/gse-sound-output-device-chooser#167]
It would be awesome if the additional bulkiness this extension adds to the system tray drop down could be removed. I think it would be nice if instead of inserting new menu options, a drop down arrow could be added to the existing microphone/speaker volume control sliders.

The drop down's functionality would be identical to the related menu options of course, but it would dramatically simplify the interface and make it feel seamlessly integrated with vanilla GNOME.

**Comment by thesmallcreeper** [comment]
Hey @PerryHugh  you might want to check my pull request. Let me know what you think about it! Thanks!

**Comment by PerryHugh** [comment]
@thesmallcreeper that is perfection, thank you for building it out!

**Comment by thesmallcreeper** [comment]
@PerryHugh Yeah also thanks for the idea. I like it! I already using it :smile:
